Header header
string description
geometry_msgs/Point point
geometry_msgs/Vector3 dimensions # size of bounding box (x, y, z)